joblib/test/test_parallel.py

```diff
@@ -883,16 +854,8 @@ def test_joblib_exception():
 
 def test_safe_function():
     safe_division = SafeFunction(division)
```

Comment:
I am under the impression that the SafeFunction class is now useless. Maybe we should add a "TODO" comment its docstring to actually check that this is really the case and post pone its deprecation / removal to a latter PR.

Reply:
In particular we need to check how `KeyboardInterrupt` are jandled in Python 3.6+ version of multiprocessing and concurrent futures.